code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -121,6 +120,8 @@ GLOBAL_LIST_EMPTY(scp173s)
 	. = ..()
 	if(isobj(loc))
 		return
+	if(length(GLOB.clients) >= 30 && !client)
+		return
```

Comment:
This'll probably cause the funny situation of "Hey, peanut just fucking went into a coma for some reason" when the pop is unstable around 29-30 but that's acceptable